SECURITY-INSIGHTS.yml

```
@@ -0,0 +1,64 @@
+header:
+  schema-version: 1.0.0
+  expiration-date: '2023-08-31T10:10:09.000Z'
+  last-updated: '2021-09-01'
+  last-reviewed: '2022-09-01'
+  commit-hash: 4dbf78ebc006ee5f668c0a74876ef8d6db9485be
+  project-url: 'https://github.com/kubernetes-sigs/kueue'
+  project-release: 0.5.1
+  changelog: 'https://github.com/kubernetes-sigs/kueue/tree/main/CHANGELOG'
+  license: 'https://github.com/kubernetes-sigs/kueue/blob/main/LICENSE'
+project-lifecycle:
+  status: active
+  roadmap: >-
+    https://github.com/kubernetes-sigs/kueue/blob/9464e42a54d67f785e2e19b8c8bbd4471cd283ad/README.md?plain=1#L98
+  bug-fixes-only: false
+  core-maintainers:
+    - 'https://github.com/alculquicondor'
+    - 'https://github.com/tenzen-y'
```

```suggestion
    - https://github.com/kubernetes-sigs/kueue/blob/main/OWNERS
```

as seen at https://github.com/argoproj/argo-cd/blob/master/SECURITY-INSIGHTS.yml#L16